src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts

```diff
@@ -0,0 +1,20 @@
+﻿namespace Windows.UI.ViewManagement {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['namespace' and 'module' are disallowed](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4878148)

Reply:
I still don't know how to resolve this one 🤔

Reply:
Don't mind those.

Reply:
Still trying to tune Codacy.  Sorry, still false positives for typescript code.